packages/amplify-e2e-core/src/utils/projectMeta.ts

```diff
@@ -41,6 +41,11 @@ function getBackendConfig(projectRoot: string) {
   return JSON.parse(fs.readFileSync(backendFConfigFilePath, 'utf8'));
 }
 
+function getCloudBackendConfig(projectRoot: string) {
```

Comment:
Are we using this anywhere other than tests?

Reply:
no, I added it for the e2e test